docs/models.md

```diff
@@ -41,50 +42,51 @@ Would be equivalent to:
 
 ```crystal
 class Bar < Granite::Base
-  adapter mysql
-    
-  field created_at : Time
-  field updated_at : Time
+  connection mysql
+
+  column created_at : Time?
+  column updated_at : Time?
 end
 ```
 
-## Custom Primary Key
+## Primary Keys
 
-For legacy database mappings, you may already have a table and the primary key is not named `id` or `Int64`.
+Each model is required to have a primary key defined.  Use the `column` macro with the `primary: true` option to denote the primary key. 
 
-Use the `primary` macro to define your own primary key
+> **NOTE:** Composite primary keys are not yet supported.
 
 ```crystal
```

Comment:
Add an example here of the standard primary key of `column id : Int64, primary: true` and then show how to define a custom id below that.

Reply:
What benefit would that bring?  Would be the same just with a different name.

Reply:
You can change this to `id` if you don't think we need two examples but it's a well known naming convention (at least in the rails world) that `id` is used for the primary key and `{model_name}_id` is used for a foreign key.  Right now in the models.md, there is no example of the recommended naming convention.